packages/preview/paper-udesc/0.0.1/typst.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "paper-udesc"
```

Comment:
If this isn't an official template you will need to have some non-descriptive prefix. See our [naming rules](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules).
Also it's common to have packages named `<prefix>-<organization>-<publication-type>`, so in this case `<non-descriptive>-udesc-paper`.